qa/L0_lifecycle/lifecycle_test.py

```diff
@@ -3493,6 +3493,79 @@ def test_delete_custom_config(self):
             except Exception as ex:
                 self.assertTrue(False, "unexpected error {}".format(ex))
 
+    def test_load_new_model_version(self):
```

Comment:
Did you ever test the scenario where we change the `version_policy` in a way that we expect to trigger certain loads, to make sure "ignoring" the version policy in the core PR doesn't break that?

Reply:
Added the test: [Expand test to cover unspecifying loaded versions](https://github.com/triton-inference-server/server/pull/7527/commits/6beab1c88382992b672b611df2ae027d6620321e)

Reply:
It is working locally, waiting on CI to rerun the L0_lifecycle with updated test script.